bodhi/client/__init__.py

```diff
@@ -787,5 +814,137 @@ def print_resp(resp, client, verbose=False):
             click.echo(caveat.description)
 
 
+@cli.group()
+def releases():
+    # Docs that show in the --help
+    """Interact with releases."""
+    # Developer Docs
+    """Manage the releases."""
+    pass  # pragma: no cover
+
+
+@releases.command(name='create')
+@handle_errors
+@add_options(release_options)
+@url_option
+def create_release(username, password, url, **kwargs):
+    """Create a release."""
+    client = bindings.BodhiClient(base_url=url, username=username, password=password)
```

Comment:
If you add the `staging_option` that would need to be given as an argument here. you can check https://github.com/fedora-infra/bodhi/blob/develop/bodhi/client/__init__.py#L269